src/v/config/tests/CMakeLists.txt

```diff
@@ -30,3 +30,9 @@ rp_test(
   LIBRARIES v::seastar_testing_main v::config
 
 )
+rp_test(
+  UNIT_TEST
+  BINARY_NAME seed_server_property_test
+  SOURCES seed_server_property_test.cc
```

Comment:
should we add this to the existing test ?

Reply:
done